tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -2,29 +2,38 @@
 
 namespace FINDOLOGIC\Export\Tests;
 
-use FINDOLOGIC\Export\Helpers\DataHelper;
+use FINDOLOGIC\Export\Data\Price;
+use FINDOLOGIC\Export\Helpers\UsergroupAwareNumericValue;
+use FINDOLOGIC\Export\Helpers\ValueIsNotNumericException;
 use FINDOLOGIC\Export\Helpers\EmptyValueNotAllowedException;
+use FINDOLOGIC\Export\Helpers\DataHelper;
 use PHPUnit\Framework\TestCase;
 
 class DataHelperTest extends TestCase
 {
     /**
      * @dataProvider emptyValueProvider
      *
-     * @param $value
-     * @param $shouldCauseException
+     * @param $value string|int value that should be checked.
```

Comment:
Data provider also has a boolean value in the test cases. `string|int|bool`

Reply:
✅